ts/features/itwallet/issuance/store/reducers/index.ts

```diff
@@ -0,0 +1,43 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import * as O from "fp-ts/lib/Option";
+import { PersistConfig, persistReducer } from "redux-persist";
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../../store/actions/types";
+import { itwStoreHardwareKeyTag } from "../actions";
+
+export type ItwIssuanceState = {
+  hardwareKeyTag: O.Option<string>;
```

Comment:
Question: should we rename this to something more generic in term of reducer where we can store our persisted data? It could include our credentials and key tags. Or do you think it's better to separate them in different reducers? 

Also, `hardwareKeyTag` could be called `integrityKeyTag` across this whole PR to keep it shorter and make it clear that this is the integrity service key tag we are storing. What do you think?

Reply:
> Question: should we rename this to something more generic in term of reducer where we can store our persisted data? It could include our credentials and key tags. Or do you think it's better to separate them in different reducers?
When we created this store we called it "ItwIssuanceState" because it will be used for everything related to issuance, both eID and other credentials. I think this name could be a good choice. Do you have any suggestion?

> Also, `hardwareKeyTag` could be called `integrityKeyTag` across this whole PR to keep it shorter and make it clear that this is the integrity service key tag we are storing. What do you think?

Renamed here: https://github.com/pagopa/io-app/pull/5861/commits/5e14ad4dd926010316fd48696a850181fb438013

Reply:
Let's keep it this way then, it makes sense to me. Thanks.